lib/install/elm.rb

```diff
@@ -12,9 +15,9 @@
           "#{Webpacker::Configuration.entry_path}/hello_elm.js"
 
 puts "Installing all elm dependencies"
-run "yarn add elm elm-webpack-loader"
-run "yarn add --dev elm-hot-loader"
-run "yarn run elm package install -- --yes"
+run "#{node_bundler} elm elm-webpack-loader"
+run "#{node_bundler} --dev elm-hot-loader"
```

Comment:
Probably wrong. `yarn add --dev` works but `node install --save --dev` should be spelled `--save-dev`.

Curiously enough `-D` looks like it should work with both (`yarn -D` is `yarn --dev` while `npm -D` is `npm --save-dev`, and calling `npm --save --save-dev` does the right thing (dev wins)).